### PR TITLE
Fixed No Players No Bots No Spectators label overlapping

### PR DESCRIPTION
**Pull request by Mailaender**
Reported by @Orb370 in https://discord.com/channels/153649279762694144/388282819371204608/1023452554878791720. I misinterpreted https://github.com/OpenRA/OpenRA/commit/0b67b5bfaea9d65ca0114b8e63bbf761759c504e#diff-3d5dfbd38868fde4ca45f20d13e0f7160d0b91552c8edb6d831b3f832bd11fa9L325 which only shows `No Players` on empty servers.

### CODE REVIEW

File: mods/common/languages/en.ftl [hunk 1]
```diff
@@ -546,7 +546,7 @@ players-label =
 
 bots-label =
     { $bots ->
-        [0] No Bots
+        [0] {""}
```
Comment (PunkPun) [inline mods/common/languages/en.ftl]: This goes against the convention. This is not related to localisation so the logic should be handled elsewhere. 
Preferably here https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Mods.Common/Widgets/Logic/ServerListLogic.cs#L418-L423

**Review by PunkPun — COMMENTED** [review]
It seems that the logic itself is bogus. 

https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Mods.Common/Widgets/Logic/ServerListLogic.cs#L389

Only shows the label when there are zero players in a lobby, meaning spectators and bots text is never used. This is also true on release.

in https://github.com/OpenRA/OpenRA/commit/0b67b5bfaea9d65ca0114b8e63bbf761759c504e#diff-3d5dfbd38868fde4ca45f20d13e0f7160d0b91552c8edb6d831b3f832bd11fa9L325
I see that spaces separating the words are gone. It's visible in Orb's report

2022-09-25_00_34_29-OpenRA_-_Tiberian_Dawn

**Comment by Mailaender** [comment]
Moved the logic back to C# and added spacings.

**Review by PunkPun — APPROVED** [review]
LGTM. 

But we still need to decide how to fix Bots and Spectators translations being redundant. Remove them or change when the label is visible?

**Comment by Mailaender** [comment]
This label seems to be redundant to the list of players. I'd like to keep this a bug fix pull request and let people who enjoy redesigning UI decide what to do with it.

**Review by dragunoff — APPROVED** [review]
The label is not very useful because it only shows up for empty servers. The information it's carrying is not entirely redundant though because you can't tell how many of the clients are bots and how many players. Approving the fix and maybe we come up with a better way to show this information - maybe in the clients list.

**Comment by PunkPun** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/88a8ab70e9c549b9d153057c90629e2554647593...67d10aeffb1f15781cf93245ebf3b4265ddfaf40)
